src/journal/meson.build

```diff
@@ -3,6 +3,8 @@
 sources = files(
         'journald-audit.c',
         'journald-audit.h',
+        'journald-client.c',
+        'journald-client.h',
```

Comment:
Not a matter, but I'd like to rename them more specific, e.g. `journald-filter.c` or so, and move `client_context_read_log_filter_patterns()` to the file.

Reply:
https://github.com/systemd/systemd/pull/24058#discussion_r927949161. I can move `client_context_read_log_filter_patterns()` there though.

Reply:
OK, file name is not a big matter for me. Feel free to keep it as is.